types/Config.d.ts

```diff
@@ -19,13 +19,23 @@ export type Config = {
   features?: Features;
   custom?: Record<string, unknown>;
   contentSigningService?: ContentSigningConfig;
+  contentProtection?: ContentProtection;
 };
 
 export type ContentSigningConfig = {
   host: string;
   drmPolicyId?: string;
 };
 
+export type ContentProtection = {
+  accessModel: 'free' | 'authfree' | 'authvod' | 'svod';
```

Comment:
We called the new setting 'freeauth': https://github.com/jwplayer/showcase-services/blob/962f7c48efcd0c9673eb5e8e06dcc4edc801707e/src/management/schemas/app_configs.py#L81

```suggestion
  accessModel: 'free' | 'freeauth' | 'authvod' | 'svod';
```